### PR TITLE
Fix: init tooltips from the app start

### PR DESCRIPTION
**Pull request by ylecuyer**
**Before**
Base on @campersau comment here : https://github.com/FredrikNoren/ungit/pull/1340#issuecomment-622092861

Tooltip are not loaded until a repo is opened

80755823-c7cd4b80-8b31-11ea-970d-9e9c7e892f68

**Fix**

Instead of relying on some hand made event to init tooltips, this PR uses the event-delegation method provided by bootstrap tooltip (based on jquery.on) to init tooltips on existing elements and new elements.

https://getbootstrap.com/docs/4.0/components/tooltips/

Screenshot_2020-05-01_23-36-05

### CODE REVIEW

File: public/source/main.js [hunk 1]
```diff
@@ -41,12 +41,9 @@ var adBlocker = require('just-detect-adblock');
       clearTimeout(id);
     };
 
-  programEvents.add(function(event) {
-    if (event.event === 'init-tooltip') {
-      $('.bootstrap-tooltip').tooltip().removeClass('bootstrap-tooltip');
```
Comment (ylecuyer) [inline public/source/main.js:46]: No need to remove the `bootstrap-tooltip` class anymore as the init process of new elements is handled by bootstrap and won't call it twice one existing tooltips

Comment (jung-kim) [reply public/source/main.js:46]: Honestly, I think we can get rid of `bootstrap` as it's footprint has been reduced.

Comment (Hirse) [reply public/source/main.js:46]: @jung-kim I'm not sure what you mean. 
Bootstrap is still used all over the place with tooltips, modals, dropdowns, and general styling.

File: public/source/main.js [hunk 2]
```diff
@@ -41,12 +41,9 @@ var adBlocker = require('just-detect-adblock');
...
-    }
+  $(document).tooltip({
+    selector: '.bootstrap-tooltip'
```
Comment (Hirse) [inline public/source/main.js:45]: Instead of the class, can/should we use `[data-toggle="tooltip"]`?

I think the class was only used (and added) so there was something to distinguish the initialized from the uninitialized tooltips.

Of course, that change could be a separate PR

Comment (campersau) [reply public/source/main.js:45]: Yes, following the bootstrap standard `[data-toggle="tooltip"]` selector would be good.
